### ISSUE TITLE
How to set up lualine to be NON-transparent ?

### ISSUE THREAD

**Issue by FullStackPete** [xiyaowong/transparent.nvim#76]
Hi there! I'm having a little trouble here... I'm currently using pywal theme with lualine. Basically everything in my editor is transparent besides terminal. I'd like to add lualine to also be non-transparent, how do I do that? My current config is below
```
{
		"xiyaowong/transparent.nvim",
		config = function()
			require("transparent").setup({
				groups = { 
					"Normal",
					"NormalNC",
					"Comment",
					"Constant",
					"Special",
					"Identifier",
					"Statement",
					"PreProc",
					"Type",
					"Underlined",
					"Todo",
					"String",
					"Function",
					"Conditional",
					"Repeat",
					"Operator",
					"Structure",
					"LineNr",
					"NonText",
					"SignColumn",
					"CursorLine",
					"CursorLineNr",
					"EndOfBuffer",
					"WinSeparator",
				},
				extra_groups = { "NeoTreeNormal", "NeoTreeNormalNC", "NeoTreeCursorLine", "" }, 
				exclude_groups = { 
					"NormalFloat", 
					"FloatBorder", 
					"Term*", 
					"winblend",
					"lualine",
					"StatusLine",
					"StatusLineNC",
				},
			})
		end,
	},
```
tldr; how to make lualine non-transparent?

**Comment by maximtrp** [comment]
@FullStackPete  Try to move `"lualine"` from `exclude_groups` to `extra_groups`.

**Comment by FullStackPete** [comment]
Also, for some reason my terminal doesn't always have background... It sometimes opens as transparent which is unwanted for me. Any tips or indications where did I do wrong? I'm using onedarkpro theme and that's it for the ui.
